### PR TITLE
Orders implementation

### LINKED ISSUE #37: As a user slightly familiar with the app, i can add orders to a particular member (saves current date/time)
So that I can track their past orders

### LINKED ISSUE #38: As a user looking to develop the business, I can check the past orders of a member
So that I can refer back to past orders for marketing/accounting purposes

### PR DESCRIPTION
**Pull request by billyhoce**
Adds functionality of tracking and adding the orders of Members. Commands added: `addorder`
close #37 close #38

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddOrderCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * Parses input arguments and creates a new AddOrderCommand object
+ */
+public class AddOrderCommandParser implements Parser<AddOrderCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddOrderCommand
+     * and returns a AddOrderCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddOrderCommand parse(String args) throws ParseException {
```
Comment (kab-dot) [inline src/main/java/seedu/address/logic/parser/AddOrderCommandParser.java:26]: Looks good and makes sense

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -38,6 +40,7 @@ public class EditCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
             + "by the index number used in the displayed person list. "
             + "Existing values will be overwritten by the input values.\n"
+            + "Orders cannot be edited this way.\n"
```
Comment (kab-dot) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:43]: Good addition

File: src/main/java/seedu/address/logic/commands/AddOrderCommand.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    /**
+     * Generates a command execution success message based on whether
+     * the order is added to or removed from
+     * {@code personToEdit}.
+     */
+    static String generateSuccessMessage(Person personToUpdate) {
+        return String.format(MESSAGE_ADD_ORDER_SUCCESS, personToUpdate.getName());
+    }
```
Comment (kokerinks) [inline src/main/java/seedu/address/logic/commands/AddOrderCommand.java:78]: Minor one: I think your javadocs `personToEdit` should be `personToUpdate`? to refer to the argument right

Comment (kokerinks) [reply src/main/java/seedu/address/logic/commands/AddOrderCommand.java:78]: or can change `personToUpdate` to `personToEdit` is up to you

**Review by kokerinks — CHANGES_REQUESTED** [review]
Other than that one comment, LGTM

File: src/main/java/seedu/address/logic/commands/AddOrderCommand.java (hunk 3 above)
Comment (billyhoce) [reply src/main/java/seedu/address/logic/commands/AddOrderCommand.java:77]: Fixed! Thanks!

**Review by kokerinks — APPROVED** [review]

**Pull request merged**